pom.xml

```diff
@@ -0,0 +1,394 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+   /*
+    *
+    *  Copyright 2017 Expedia, Inc.
+    *
+    *     Licensed under the Apache License, Version 2.0 (the "License");
+    *     you may not use this file except in compliance with the License.
+    *     You may obtain a copy of the License at
+    *
+    *         http://www.apache.org/licenses/LICENSE-2.0
+    *
+    *     Unless required by applicable law or agreed to in writing, software
+    *     distributed under the License is distributed on an "AS IS" BASIS,
+    *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    *     See the License for the specific language governing permissions and
+    *     limitations under the License.
+    *
+    */
+  -->
+
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.expedia.www</groupId>
+    <artifactId>haystack-trends</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+    <name>haystack-trends</name>
+    <description>This is the parent module for haystack-trends</description>
+
+    <licenses>
+        <license>
+            <name>The Apache License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+        </license>
+    </licenses>
+
+    <modules>
+        <module>span-timeseries-transformer</module>
+        <module>timeseries-aggregator</module>
+        <module>commons</module>
+    </modules>
+
+    <properties>
+
+        <!--Dependency Properties -->
+
+        <pegdown.version>1.6.0</pegdown.version>
+        <scalatest.version>3.0.3</scalatest.version>
+        <slf4j.version>1.7.25</slf4j.version>
+        <kafka.version>0.11.0.0</kafka.version>
+        <junit.version>4.12</junit.version>
+        <typesafe-config.version>1.3.1</typesafe-config.version>
+        <logback.version>1.2.3</logback.version>
+        <metrics-core.version>3.0.2</metrics-core.version>
+        <protobuf.version>3.3.1</protobuf.version>
+        <protobuf-maven-plugin.version>3.3.0.1</protobuf-maven-plugin.version>
```

Comment:
move this with the other plugin versions

Reply:
done